const.go

```diff
@@ -13,4 +13,8 @@ const (
 
 	// MaxSatoshi is the maximum transaction amount allowed in satoshi.
 	MaxSatoshi = 21e6 * SatoshiPerBitcoin
+
+	// Satoshi is the base bitcoin monetary unit.
+	// A single amount is equal to 1e-8 of a bitcoin.
```

Comment:
Constant definitions show up in the godoc documentation, so I'm not sure the second comment line adds anything.  It doesn't hurt to have it either I suppose, but I don't think it's providing any additional information.

Also, what do you think of naming the constant `BitcoinPerSatoshi`.  It's a little more consistent with the other constants and I think less confusing in that it conveys it's a multiplier.  To me, the name `Satoshi` and and of itself is the value `1` by virtue of it being the base unit.